react-time-track/src/views/edit-user.js

```diff
@@ -0,0 +1,67 @@
+/** @jsx jsx */
+import React from "react";
+import {jsx} from "@emotion/core";
+import {navigate} from "@reach/router";
+import {Li, Title, UserCard, Card, Button} from "../components/ui";
+import {getUser, editUser} from "../services/user";
+
+function EditUser({user_id}) {
+    const [user, setUser] = React.useState({
+			name: "",
+			email: "",
+			role: "",
+			rate: 0
+		});
+
+    React.useEffect(()=> {
+			console.log(user_id)
+      getUser(user_id)
+      .then(data => {setUser(data)
+    	});
+		}, []);
+
+	React.useEffect(()=> {
+		console.log(user);
```

Comment:
Same for this whole useEffect, it is used only for the console log, rigth?

Reply:
Thanks for notice them. I'll replace it right now.